docs/csharp/programming-guide/classes-and-structs/local-functions.md

```diff
@@ -40,9 +40,12 @@ Note that all local variables that are defined in the containing member, includi
 Unlike a method definition, a local function definition cannot include the following elements:
 
 - The member access modifier. Because all local functions are private, including an access modifier, such as the `private` keyword, generates compiler error CS0106, "The modifier 'private' is not valid for this item."
- 
+
 - The [static](../../language-reference/keywords/static.md) keyword. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
 
+  > [!NOTE]
+  > Starting with C# 8.0, `static` local functions are valid. A `static` local function cannot access variables from the containing method or instance fields.
```

Comment:
I would reverse this note by removing line 44, and changing the note text as follows:

```suggestion
  > Prior to C# 8.0, local functions cannot include the `static` modifier. Including the `static` keyword generates compiler error CS0106, "The modifier 'static' is not valid for this item."
```

Reply:
@BillWagner, Because removing line 44 would make the bullet list have only one item. I've made combined this item with the previous sentence.